XMonad/Util/ClickableWorkspaces.hs

```diff
@@ -0,0 +1,71 @@
+-------------------------------------------------------------------------------
+-- |
+-- Module      :  XMonad.Util.ClickableWorkspaces
+-- Copyright   :  (c) Geoff deRosenroll <geoffderosenroll@gmail.com>
+-- License     :  BSD3-style (see LICENSE)
+--
+-- Maintainer  :  Geoff deRosenroll <geoffderosenroll@gmail.com>
+-- Stability   :  unstable
+-- Portability :  unportable
+--
+-- Provides @clickablePP@, which when applied to the PP pretty-printer used by
+-- the "XMonad.Hooks.DynamicLog" hook, will make the workspace tags clickable in
+-- XMobar (for switching focus).
+--
+-----------------------------------------------------------------------------
+
+module XMonad.Util.ClickableWorkspaces (
+  -- * Usage
+  -- $usage
+  clickablePP
+  ) where
+
+import XMonad
+import XMonad.Util.WorkspaceCompare (getWsIndex)
+import XMonad.Hooks.DynamicLog (PP(..))
+
+-- $usage
+-- However you have set up your PP, apply @clickablePP@ to it, and bind the result
+-- to "XMonad.Hooks.DynamicLog"\'s dynamicLogWithPP like so:
+-- 
+-- > myPP xmprocs = xmobarPP { ... }
+-- > myLogHook xmprocs = (clickablePP . myPP) xmprocs >>= dynamicLogWithPP
```

Comment:
What is `xmprocs`? My loghook is without arguments, and so is the one in [DynamicLog usage help](https://hackage.haskell.org/package/xmonad-contrib-0.16/docs/XMonad-Hooks-DynamicLog.html).

Reply:
Ah sorry I guess I so things a bit weird, I can remove it from the example to avoid confusion. It is a list of xmobar processes that I give to ppOutput in my case.

Reply:
Well a more ordinary usage that's compatible with what DynamicLog's usage suggest would definitely be welcome. :-)